doc/nrf/libraries/security/trusted_storage.rst

```diff
@@ -52,10 +52,10 @@ Choices and backends
 The trusted storage library comes with two choices:
 
 * ``TRUSTED_STORAGE_BACKEND`` that defines a backend for handling of encryption, authentication or other means of validation of the stored data.
-  It is responsible for what is happening with the data before and after they are read from or written to non-volatile storage.
-* ``TRUSTED_STORAGE_STORAGE_BACKEND`` that defines a backend that handles how the data are written to non-volatile storage.
+  It is responsible for what is happening with the data before and after they are read from or written to a non-volatile memory storage.
```

Comment:
Sounds a bit wrong to me, I would rather just have `to non-volatile memory`.
The name ZMS had to be chosen to differentiate itself from the already existing alternatives, but we don't talk about non-volatile memory storage. Just non-volatile memory.

Reply:
Reverting back to non-volatile storage, as this was written initially.